CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [0.18.12] - 2024-05-01
+
+- Fixed a bug in the `normalise_session_scope` util function that caused it to remove leading dots from the scope string.
```

Comment:
i thought this would be a part of the duplicate cookie release. So no need for version change. Instead, make this on top of the other PR (https://github.com/supertokens/supertokens-python/pull/485)